src/components/car/CarComponent.tsx

```diff
@@ -35,14 +36,11 @@ export default class CarComponent extends React.Component<Props, State> {
     const style = computeStyleSheet();
     const { car, selected, navigation } = this.props;
     const carUsers = car?.carUsers ?? [];
-    const defaultCarUser = carUsers.length === 1 ? carUsers[0] : carUsers?.find((carUser) => carUser.default === true);
+    const defaultCarUser = carUsers.find((carUser) => carUser.default);
     const defaultCarUserName = Utils.buildUserName(defaultCarUser?.user);
     const otherUserCount = Math.max(carUsers.length - 1, 0);
-    const carCatalog = car?.carCatalog;
-    const carMake = carCatalog?.vehicleMake ?? '';
-    const carModel = carCatalog?.vehicleModel ?? '';
-    const carModelVersion = carCatalog?.vehicleModelVersion ?? '';
-    const carFullName = carMake + ' ' + carModel + ' ' + carModelVersion;
+    const carFullName = Utils.buildCarCatalogName(car?.carCatalog);
+    const userIDs = carUsers.map((userCar: UserCar) => userCar?.user?.id);
```

Comment:
Here you can have empty record if no user is provided for the car.
I did the change.